### ISSUE TITLE
[Question]: Backtrailing slashes for directories

### ISSUE THREAD

**Issue by NightSling** [abgox/PSCompletions#39]
### Before Start

- [X] You are using the latest version of PSCompletions.
- [X] You have read this repository's README file.
- [X] You have search for related issues in [Issues](https://github.com/abgox/PSCompletions/issues).
- [X] You have visit https://pscompletions.pages.dev/

### What Powershell version are you using?

PowerShell

### What system are you using?

Windows

### Steps to reproduce

Use the completion menu to access a directory. 

```bash
> ls
Directory
> cd Dir<TAB> (menu opens and select Directory)
```

### What is expected?

Add a trailing backslash to directories when completion occurs.
```bash
> cd .\Directory\
```

### What is actually happening?

No trailing backslash
```bash
> cd .\Directory
```

### Any additional comments?

(This is more like Q/A Issue rather than Bug report, please let me know how I can fix this.)

**Comment by abgox** [comment]
- Path completion is provided by PowerShell's built-in TabExpansion2.
- The path completion returned by `TabExpansion2` does not come with `\` , you may need to add it manually.

**Comment by NightSling** [comment]
Is there any way to add it? 
When I normally use Tab on PowerShell for directory path completion, it does add `\` to the end.

**Comment by abgox** [comment]
- It will be fixed in `v5.0.6`.
- Please wait.
